web/frontend/src/components/slam.vue

```diff
@@ -95,6 +104,76 @@ const fetchSLAMPose = (name: string): Promise<commonApi.Pose> => {
   });
 };
 
+const executeMove = async () => {
+  moveClick = false;
+
+  const req = new motionApi.MoveOnMapRequest();
+
+  /*
+   * set request name
+   * here we set the name of the motion service the user is using
+   */
+  req.setName('builtin');
+
+  const value = await fetchSLAMPose(props.name);
+  // set pose in frame
+  const sentPose = new commonApi.Pose();
+  sentPose.setX(Math.abs(threeJPos.x - value.getX()));
+  sentPose.setY(Math.abs(threeJPos.z - value.getY()));
```

Comment:
[note] @nfranczak here is a place where we are using the Y/Z values due to the frame/plane transfer. Please look into the expected plane we want to represent this in so the SLAM team can update cartographer position (and maybe pointcloud) to correspond to this.